### PR TITLE
Adjust MetalLB E2E tests run to the new tests configuration in MetalLB repo

### PR DESCRIPTION
**Pull request by sabinaaledort**
MetalLB E2E tests are now creating and deleting MetalLB configuration when running.

### CODE REVIEW

**Comment by sabinaaledort** [comment]
waiting for https://github.com/metallb/metallb/pull/1008

**Comment by sabinaaledort** [comment]
/retest
failed to create the OCP cluster (unrelated to my change)

**Comment by sabinaaledort** [comment]
/retest
failed to create the OCP cluster (unrelated to my change)

**Comment by oribon** [comment]
lgtm

File: metallb/start_frr.sh [hunk 1]
```diff
@@ -1,37 +0,0 @@
...
-set -e
-
-## configure frr
-# cleanup frr directory
-[[ -d ${metallb_dir}/frr ]] &&
-	sudo rm -rf ${metallb_dir}/frr
-
-# add firewalld rules
-sudo firewall-cmd --zone=libvirt --permanent --add-port=179/tcp
-sudo firewall-cmd --zone=libvirt --add-port=179/tcp
```
Comment (oribon) [inline metallb/start_frr.sh:23]: will it work without this (and port 180 for the other container)?

Comment (sabinaaledort) [reply metallb/start_frr.sh:23]: you are right, we need that, i will add it

File: 06_create_cluster.sh [hunk 2]
```diff
@@ -49,25 +49,24 @@ if [[ -n "${APPLY_EXTRA_WORKERS}" ]]; then
...
+if [[ ! -z "${METALLB_MODE}" ]]; then
 
 	if [[ -z ${METALLB_IMAGE_BASE} ]]; then
 		export METALLB_IMAGE_BASE=$(\
 			jq -r .references.spec.tags[0].from.name ${OCP_DIR}/release_info.json | sed -e 's/@.*$//g')
 		export METALLB_IMAGE_TAG="metallb"
 	fi
 
-	if [[ ${ENABLE_METALLB_MODE} == "l2" ]]; then
+	if [[ ${METALLB_MODE} == "L2" ]]; then
```
Comment (fedepaol) [inline 06_create_cluster.sh:60]: There should no need for l2 / bgp mode anymore

**Comment by fedepaol** [comment]
/lgtm
cc @russellb

**Comment by sabinaaledort** [comment]
/retest
origin-tests failed (unrelated to my change)

**Comment by sabinaaledort** [comment]
/retest
images mirroring failed (unrelated to my change)

**Comment by sabinaaledort** [comment]
/retest
origin-tests failed (unrelated to my change)

**Comment by sabinaaledort** [comment]
/retest
failed to create OCP cluster (unrelated to my change)

**Comment by sabinaaledort** [comment]
/retest
origin-tests failed (unrelated to my change)

**Pull request merged**